File: 01 - Automata.Presentation/Automata.UI/src/app/entities/method-parameters/common/baseMethodOut.ts
import { OperationResult } from '../../../entities/entities.module';

export class BaseMethodOut {
    private _operationResult: OperationResult;

    get OperationResult(): OperationResult { return this._operationResult };
    set OperationResult(value: OperationResult) { this._operationResult = value };
}

File: 03 - Automata.Business/Automata.MethodParameters/BaseMethodOut.cs
using Automata.Entities.Common;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Automata.MethodParameters
{
    public class BaseMethodOut
    {
        public OperationResult OperationResult { get; set; }

        public BaseMethodOut()
        {
            this.OperationResult = OperationResult.Error;
        }
    }
}
